tests/schema/test_validate.py

```diff
@@ -147,8 +142,8 @@ def test_group_rule_on_nulls(
 ) -> None:
     # The schema is violated because we have multiple "b" values for the same "a" value
     df = df_type({"a": [None, None], "b": [1, 2]})
-    with pytest.raises(RuleValidationError):
-        schema.validate(df, cast=True)
+    with pytest.raises(plexc.ComputeError, match=r"1 rules failed validation"):
```

Comment:
Does this have to be a generic ComputeError? I saw `validation_error.rs`, which seemed like there would still be a distinct `RuleValidationError`